app/controllers/docs_controller.rb

```diff
@@ -20,13 +20,53 @@ class DocsController < ActionController::API
     key :consumes, ['application/json']
     key :produces, ['application/json']
   end
+
```

Comment:
@Br3nda do we need all this commented code?

Reply:
removed it now